### PR TITLE
add KvAPI into cdcpb.proto for rawkv cdc

### PR DESCRIPTION
**Pull request by zeminzhou**
To support RAWKV and TXNKV, a KvAPI field was added to cdcpb.proto。

There are three types of data in TiKV, namely TiDB, RawKV, and TxnKV. When the data in the TiKV cluster is synchronized to the downstream cluster through the CDC, the CDC needs to provide the KvAPI field to indicate the type of data it wants to synchronize.

### CODE REVIEW

File: proto/cdcpb.proto [hunk 1]
```diff
@@ -150,6 +150,14 @@ message ChangeDataRequest {
...
         NotifyTxnStatus notify_txn_status = 10;
     }
+
+    enum KvAPI {
+       TiBD = 0;
+       RawKV = 1;
+       TxnKV = 2;
+    }
+
+    KvAPI kv_api = 11;
```
Comment (pingyu) [inline proto/cdcpb.proto:160]: Suggest add comments about what this field is used for.

**Comment by pingyu** [comment]
Suggest briefly introduce what is "rawkv cdc", and what the new field is used for, in PR description.

**Comment by zeminzhou** [comment]
> Suggest briefly introduce what is "rawkv cdc", and what the new field is used for, in PR description.

Done

File: proto/cdcpb.proto [hunk 2]
```diff
@@ -150,6 +150,15 @@ message ChangeDataRequest {
         // min_commit_ts so that the resolved_ts can be advanced.
         NotifyTxnStatus notify_txn_status = 10;
     }
+
+    // Used for CDC to indicate the type of data it wants to synchronize.
```
Comment (pingyu) [inline proto/cdcpb.proto:154]:
```suggestion
    // KvAPI specifies to capture data written by different KV API.
    // See more details in https://github.com/tikv/rfcs/blob/master/text/0069-api-v2.md.
```

**Comment by pingyu** [comment]
> /cc @overvenus @andylokandy

PTAL, thanks~

**Review by pingyu — APPROVED** [review]
LGTM

**Review by disksing — APPROVED** [review]

**Pull request merged**